### DISCUSSION TITLE
Missing latest releases for several projects, what's happening to GitHub?

### DISCUSSION THREAD

**Discussion by eleius** [community/community#23516] [Projects and Issues]
<p>Hello. I noticed several projects like notepad-plus-plus, sandboxie-plus, yt-dlp, magisk and others, are missing the last 1 or 2 releases (e.g. reported <a href="https://github.com/notepad-plus-plus/notepad-plus-plus/issues/11654" rel="noopener nofollow ugc">here</a>). Is this a known issue and is there a page somewhere tracking it?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>there are similar threads, like this one</p>
<aside class="quote quote-modified" data-post="1" data-topic="2244">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/latest-release-isnt-the-real-latest/2244">Latest release isn't the real latest</a> <a class="badge-wrapper  bullet" href="/c/github-help/how-to-use-git-and-github/21"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #ffd1ac;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Learn how to start building, shipping, and maintaining software with GitHub (and Git) in this category.">How to use Git and GitHub</span></a>
  </div>
  <blockquote>
    Hi, 
I am facing an issue for now several weeks. 
Context 
We have 3 branches to maintain (3 versions of a Software). 
Those versions are 2.8.x, 18.10.x, 19.04.x 
I’ve released two new versions on the 18.10.x and 19.04.x in order to do that I’ve started this way 


Tag on branch 18.10.x + release on GitHub 


Tag on branch 19.04.x + release on GitHub 


I was expecting the 19.04.x version to be tagged as the latest but the 18.10.x is tagged as the latest. 
Any idea on how to prevent this kind of…
  </blockquote>
</aside>

<p>this is the feedback product team, maybe it’s a bug that they can fix</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/github/feedback/discussions">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/github/feedback/discussions" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/47092464/181077498-02fe1f9b-4205-41bf-8842-ee5025227c25.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/d/7d9b1e42bb87972350d5e7b32aa0e82622c98335_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/d/7d9b1e42bb87972350d5e7b32aa0e82622c98335_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/7/d/7d9b1e42bb87972350d5e7b32aa0e82622c98335.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/d/7d9b1e42bb87972350d5e7b32aa0e82622c98335_2_10x10.png"></div>

<h3><a href="https://github.com/github/feedback/discussions" target="_blank" rel="noopener">Discussions · github/feedback</a></h3>

  <p>Public feedback discussions for: GitHub for Mobile, GitHub Discussions, GitHub Codespaces, GitHub Sponsors, GitHub Issues and more! - Discussions · github/feedback</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>you can encourage all those interested to upvote it</p>

**Comment by cp2004** [comment]
<p>Apparently they are aware and working on it. It is a strange issue.</p>
<aside class="quote" data-post="2" data-topic="249861">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/releases-vanishing/249861/2">Releases vanishing</a> <a class="badge-wrapper  bullet" href="/c/github-help/48"><span class="badge-category-bg" style="background-color: #6a737d;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Ask questions about your GitHub account and the core features of GitHub.">GitHub Help</span></a>
  </div>
  <blockquote>
    I created a (private) ticket yesterday, GitHub have acknowledged the problem and are working to fix it. 
The workaround to make the releases show up again on the list: 

editing the release and clicking on Update release without making any changes should force a re-sync and the missing release will re-appear.
  </blockquote>
</aside>
